rust/Cargo.toml

```diff
@@ -82,3 +82,7 @@ strip = "none"
 debug = "full"
 split-debuginfo = "packed"
 strip = "none"
+
+# Override build settings for the relay, so we can capture flamegraphs
+[profile.release.package.firezone-relay]
+debug = "full"
```

Comment:
Would it be more appropriate to [use the `bench` profile](https://doc.rust-lang.org/cargo/reference/profiles.html#bench) for this instead?

Reply:
Hmm, I don't know if I can build the regular binary using the bench profile. Probably but it will be a bit quirky.

Also, I'd ideally like to be able to take what is running in production and benchmark that and not rebuild something.

i.e. use the same docker container etc